Automattic/wp-calypso
# Jetpack Manage: Onboarding - Add Dashboard Walkthrough guided tour

Resolves https://github.com/Automattic/jetpack-manage/issues/158 

## Proposed Changes

* This PR adds the Dashboard Walkthrough guided tour, as described in https://github.com/Automattic/jetpack-manage/issues/158

## Testing Instructions

* You must be an agency to be able to this this PR: 2c49b-pb. Make sure to switch it back to our original type after testing. Or not. Your call!
* Switch to this branch. (`git checkout add/jp-manage/158-dashboard-walkthrough-onboarding-tour` and yarn start-jetpack-cloud).
* Open http://jetpack.cloud.localhost:3000/, and you'll be redirected to the /dashboard.
* Open the Plugin Management page
* While in the dashboard, append the URL with `?tour=dashboard-walkthrough`
* You should see the 8 steps of the plugin management tour.
* Once you finish the tour, to run it again you must erase the preference `jetpack-cloud-sites-overview-dashboard-walkthrough-tour`. You can do this by hovering in the bottom right button called `Preferences`, then clicking over the `X` next to the preference name.
* 
![image](https://github.com/Automattic/wp-calypso/assets/37049295/e8d96bab-42d8-4254-a682-29c286e3ea78)

Screenshot of all steps (to help our translators):

<img width="332" alt="image" src="https://github.com/Automattic/wp-calypso/assets/37049295/dc877ca4-bcce-470d-9b9a-c3169f66e92b">
<img width="332" alt="Screenshot 2023-12-22 at 17 01 10" src="https://github.com/Automattic/wp-calypso/assets/16754605/86ab2eb2-7084-4441-acb4-bf10e277b771">


<img width="358" alt="image" src="https://github.com/Automattic/wp-calypso/assets/37049295/7c85bb8f-afdc-4de4-abaf-ce97dd452fb9">

<img width="399" alt="image" src="https://github.com/Automattic/wp-calypso/assets/37049295/3ff3b191-7c63-47c5-941e-cf2ab46cf894">

<img width="354" alt="image" src="https://github.com/Automattic/wp-calypso/assets/37049295/f67ba087-4110-4170-a40b-ab88c55bd904">


<img width="332" alt="Screenshot 2023-12-22 at 16 59 52" src="https://github.com/Automattic/wp-calypso/assets/16754605/dd2a5be3-7285-4fec-8e96-899b06505fa7">


<img width="332" alt="image" src="https://github.com/Automattic/wp-calypso/assets/37049295/8429979c-2722-4108-8754-7a3443b9db8b">

<img width="400" alt="image" src="https://github.com/Automattic/wp-calypso/assets/37049295/7ac051e3-90be-4db5-a9e7-db40118cd9ef">



## Pre-merge Checklist
